Automattic/jetpack
# CLI: Update `generate` to handle github actions better.

Fixes #

#### Changes proposed in this Pull Request:

When creating a new github action using `jetpack generate`, we need some more info in `extra.autotagger` so it creates the "v1" tag as well as "v1.0.0". 

#### Jetpack product discussion


https://github.com/Automattic/jetpack/pull/19818#pullrequestreview-690889334

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* run `jetpack generate github-action` 
* When you're prompted for "Will this project require a mirror repo?" say Yes.
* When prompted for "Configure mirror repo to create new tags automatically (based on CHANGELOG.md)?" say Yes.
* After the project is created, check the project's composer.json and make sure that `extra.autotagger` is set to `{ "major": true }`
* Make sure non github-action types have `extra.autotagger` just set to `true`
* Answering negative to the autotagger question should set everything to false for all project types.
